CHANGELOG

```diff
@@ -1,6 +1,11 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## [3.8.0] - 2023-06-13
+# Added
+- Introducing new option `create_integration_branches`
```

Comment:
We also introduce a general setting about it. I would not focus on the name of the option, more in the change of behavior here.